src/test/java/com/salesforce/dynamodbv2/ScanTest.java

```diff
@@ -135,6 +138,45 @@ void scanAll(TestArgument testArgument) {
         });
     }
 
+
+    @ParameterizedTest(name = "{arguments}")
+    @ArgumentsSource(DefaultArgumentProvider.class)
+    void scanAllTenants(TestArgument testArgument) {
+        MT_CONTEXT.setContext(null);
+        String startTableName = null;
+        List<String> tableNames;
+        int maxPages = 5;
+        int attemptCount = 0;
+        do {
+            List<String> allTableNames = testArgument.getAmazonDynamoDb().listTables(startTableName).getTableNames();
+
+            //filter scan to just tables this mt dymamo strategy manages
```

Comment:
-> "// filter scan to just tables managed by this mt-dynamo strategy"

Reply:
(FYI: Intellij's spell checker should catch "dymamo" (though it probably won't complain about "mt dynamo" (as opposed to "my-dynamo"))—and other potential issues)